static/app/views/settings/organizationAuditLog/index.spec.tsx

```diff
@@ -144,7 +144,7 @@ describe('OrganizationAuditLog', function () {
     expect(
       screen.getByText(
         textWithMarkupMatcher(
-          `Disabled dynamic sampling priority "Prioritize key transactions" in project ${project.slug}`
+          `Disabled retention priority "boostKeyTransactions" in project ${project.slug}`
```

Comment:
shouldn't we display the translation instead of the key? the key is not visible to the users

Reply:
nevermind... this is audit logs... but in any case it would be nice if we had the correct translations here

Reply:
yeah I agree, changed it so that we use labels 👍